src/annotator/config/test/index-test.js

```diff
@@ -51,24 +67,117 @@ describe('annotator/config/index', function () {
     });
   });
 
-  ['appType', 'openSidebar', 'subFrameIdentifier'].forEach(function (
-    settingName
-  ) {
-    it(`reads ${settingName} from the host page, even when in a browser extension`, function () {
-      getConfig('all', 'WINDOW');
-      assert.calledWith(
-        fakeSettingsFrom().hostPageSetting,
-        settingName,
-        sinon.match({ allowInBrowserExt: true })
-      );
+  describe('browser extension', () => {
+    context('when client is loaded from the browser extension', function () {
+      it('returns only config values where `allowInBrowserExt` is true or the defaultValue if provided', () => {
```

Comment:
Since a `defaultValue` is set for every setting, the case where a config value is not returned is not exercised here. A more accurate description of what is tested currently would be "reads only config values where `allowInBrowserExt` is true, otherwise returns `defaultValue`".